examples/PhasorDynamics/Tiny/TwoBus/Ieeet1/TwoBusIeeet1.cpp

```diff
@@ -0,0 +1,337 @@
+  data.bus_fault[0].parameters[BusFaultParameters::state0] = false;
+
+  // ------------- MODEL GROUP ------------------
+
+  // Set generator data
+  data.genrou.resize(1);
+
+  data.genrou[0].parameters[GenrouParameters::p0]    = 1.;
+  data.genrou[0].parameters[GenrouParameters::q0]    = 0.05013;
+  data.genrou[0].parameters[GenrouParameters::H]     = 3.;
+  data.genrou[0].parameters[GenrouParameters::D]     = 0.;
+  data.genrou[0].parameters[GenrouParameters::Ra]    = 0.;
+  data.genrou[0].parameters[GenrouParameters::Tdop]  = 7.;
+  data.genrou[0].parameters[GenrouParameters::Tdopp] = .04;
+  data.genrou[0].parameters[GenrouParameters::Tqopp] = .05;
+  data.genrou[0].parameters[GenrouParameters::Tqop]  = .75;
+  data.genrou[0].parameters[GenrouParameters::Xd]    = 2.1;
+  data.genrou[0].parameters[GenrouParameters::Xdp]   = 0.2;
+  data.genrou[0].parameters[GenrouParameters::Xdpp]  = 0.18;
+  data.genrou[0].parameters[GenrouParameters::Xq]    = 0.5;
+  data.genrou[0].parameters[GenrouParameters::Xqp]   = 0.5;
+  data.genrou[0].parameters[GenrouParameters::Xqpp]  = 0.18;
+  data.genrou[0].parameters[GenrouParameters::Xl]    = 0.15;
+  data.genrou[0].parameters[GenrouParameters::S10]   = 0.;
+  data.genrou[0].parameters[GenrouParameters::S12]   = 0.;
+
+  // Governor
+  data.gov.resize(1);
+
+  // Exciter
+  data.exciter.resize(1);
+
+  data.exciter[0].parameters[Exciter::Ieeet1Parameters::Tr]      = 0.001; // (BUG: Nonfunctional if Tr = 0)
+  data.exciter[0].parameters[Exciter::Ieeet1Parameters::Ka]      = 50.;
+  data.exciter[0].parameters[Exciter::Ieeet1Parameters::Ta]      = 0.04;
+  data.exciter[0].parameters[Exciter::Ieeet1Parameters::Ke]      = -0.06;
+  data.exciter[0].parameters[Exciter::Ieeet1Parameters::Te]      = 0.6;
+  data.exciter[0].parameters[Exciter::Ieeet1Parameters::Kf]      = 0.09;
+  data.exciter[0].parameters[Exciter::Ieeet1Parameters::Tf]      = 1.46;
+  data.exciter[0].parameters[Exciter::Ieeet1Parameters::Vrmin]   = -1.;
+  data.exciter[0].parameters[Exciter::Ieeet1Parameters::Vrmax]   = 1.;
+  data.exciter[0].parameters[Exciter::Ieeet1Parameters::E1]      = 2.8;
+  data.exciter[0].parameters[Exciter::Ieeet1Parameters::E2]      = 3.373;
+  data.exciter[0].parameters[Exciter::Ieeet1Parameters::Se1]     = 0.04;
+  data.exciter[0].parameters[Exciter::Ieeet1Parameters::Se2]     = 0.33;
+  data.exciter[0].parameters[Exciter::Ieeet1Parameters::Ispdlim] = 0.;
+
+  // -------------- END MODEL DATA ------------------
+
+  // Manually add components
+  // This is a workaround since signal connections are not implemented in parser
+
+  // Create buses
+  auto* bus0 = BusFactory<scalar_type, index_type>::create(data.bus[0]);
+  auto* bus1 = BusFactory<scalar_type, index_type>::create(data.bus[1]);
+
+  // Create signal nodes
+  auto* omega = new signal_type(data.signal[0]);
+  auto* pmech = new signal_type(data.signal[1]);
+  auto* efd   = new signal_type(data.signal[2]);
```

Comment:
these five variables are leaked. you can fix this partly by stack-allocating `omega`, `pmech`, `efd` but since you're using `BusFactory`, you need to manually free `bus0` and `bus1`.

Reply:
we have address sanitizer support (and undefined behavior sanitizer support) in gridkit. you can enable it by setting the `GRIDKIT_ENABLE_ASAN` cmake option to `yes`